src/components/questions/AudioRecorder.tsx

```diff
@@ -207,7 +208,7 @@ const AudioRecorder = (props: {
               disabled={isDisabled}
               activeOpacity={0.5}
               accessible={true}
-              accessibilityLabel={'Spraakopname afspelen knop'}
+              accessibilityLabel="Spraakopname afspelen"
```

Comment:
Of overal knop gebruiken of overal niet

Reply:
@milovdpas Besproken tijdens call 
@wwessel01 gefixt